src/frontend/src/optimizer/plan_node/mod.rs

```diff
@@ -279,9 +279,22 @@ impl GenericPlanRef for PlanRef {
     }
 }
 
-impl dyn PlanNode {
+pub trait Explainable {
```

Comment:
According to Rust's convention, I think this should be called `Explain` (no `-able`)